packages/playwright/src/reporters/html.ts

```diff
@@ -602,17 +602,10 @@ type JsonAttachment = {
 };
 
 function stdioAttachment(chunk: Buffer | string, type: 'stdout' | 'stderr'): JsonAttachment {
-  if (typeof chunk === 'string') {
-    return {
-      name: type,
-      contentType: 'text/plain',
-      body: chunk
-    };
-  }
   return {
```

Comment:
The only case that I can think of where it may be useful to see original binary output is a tool that produces e.g. an image and prints the bytes to stdout. And even in that case the users should store the output to an attachment manually rather than piping it to playwright's.

Reply:
Thanks, that makes sense!